### PR TITLE
sample-cluster-java port to 2.6/typed

### PR DESCRIPTION
**Pull request by johanandren**
All three samples ported to be as 1:1 as possible with Scala sample. Compiles and runs.

Readme and multi-jvm still left to deal with.

Not sure if we should keep multi-jvm as that was in Scala, feels bad to push that on a Java Akka-newcomer. Otoh perhaps that is good to test that the samples actually works.

### CODE REVIEW

**Comment by patriknw** [comment]
Good question about the Scala multi-jvm tests. Maybe just include multi-jvm in this sample. In the cqrs sample we show how to cluster integration testing without multi-jvm. Good to have alternatives to point at.

File: akka-sample-cluster-java/README.md [hunk 1]
```diff
@@ -21,27 +21,27 @@ The small program together with its configuration starts an ActorSystem with the
...
 
-To run this sample, type `sbt "runMain sample.cluster.simple.SimpleClusterApp"` if it is not already started. If you are using maven, use `mvn compile exec:java -Dexec.mainClass="sample.cluster.simple.SimpleClusterApp"` instead. To pass parameters to maven add `-Dexec.args="..."`.
+To run this sample, type `sbt "runMain sample.cluster.simple.App"` if it is not already started. If you are using maven, use `mvn compile exec:java -Dexec.mainClass="sample.cluster.simple.App"` instead. To pass parameters to maven add `-Dexec.args="..."`.
 
 `SimpleClusterApp` starts three actor systems (cluster members) in the same JVM process. It can be more interesting to run them in separate processes. Stop the application and then open three terminal windows.
 
 In the first terminal window, start the first seed node with the following command:
 
-    sbt "runMain sample.cluster.simple.SimpleClusterApp 2551"
+    sbt "runMain sample.cluster.simple.App 2551"
```
Comment (patriknw) [inline akka-sample-cluster-java/README.md]: mvn pom.xml and run instructions, you can use this commit as template: https://github.com/akka/akka-samples/pull/141/commits/b54f1b608d38d4f1c49d6df2a023a8f4d256f5e5

Comment (johanandren) [reply akka-sample-cluster-java/README.md]: There already is a pom.xml in this sample, I'll update the readme references.

File: akka-sample-cluster-java/src/main/java/sample/cluster/stats/App.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class App {
+
+  static final ServiceKey<StatsService.ProcessText> STATS_SERVICE_KEY =
```
[the remark this answers is not in the document]
Comment (johanandren) [reply akka-sample-cluster-java/src/main/java/sample/cluster/stats/App.java]: Used in multi-jvm test

**Pull request merged**